katas/content/multi_qubit_systems/index.md

```diff
@@ -17,15 +17,17 @@ If you are not familiar with single-qubit systems, we recommend that you complet
 
 **What you should know to start working on this kata:**
 
-- Basic single-qubit gates
+- Basic knowledge of single-qubit gates
```

Comment:
Together with the header this reads as "you should know basic knowledge of single-qubit gates", that doesn't sound very well. What is the issue with the current phrasing? I'd keep it as is

Reply:
Right, I hadn't considered the prerequisites as a follow-up to the first sentence. The reason behind including "basic knowledge" is because it's recommended to state the knowledge level, so the user can measure if they're ready, and "basic gates" isn't very specific. I'll change back every entry for "basic concepts"